Dockerfile

```diff
@@ -26,7 +26,6 @@ RUN pip install -U pip wheel
 COPY requirements /pip/requirements/
 RUN cd /pip && \
     pip install --build ./build --cache-dir ./cache \
-        --find-links https://pyrepo.addons.mozilla.org/wheelhouse/ \
```

Comment:
well, technically it should work. I'm not sure what's going on but I wanted to get this in just to unblock others. Also, my bad for not re-building the image after that change :dart: